cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -569,15 +583,37 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
                 .build();
     }
 
+    private static String converttoformat(Map<String, Object> map) {
```

Comment:
We want to avoid manually building json/yaml structures ourselves (it is prone to errors). If we cannot avoid this by just constructing the configMap directly from the load before after map we should be using Jackson to create any json/yaml strings.